chest_changes/init.lua

```diff
@@ -0,0 +1,16 @@
+minetest.override_item("tsm_chests:chest", {
+	on_rightclick = function(pos)
+		spooky_effects.spawn_ghost(pos)
+	end,
```

Comment:
Have you tested whether overriding `on_rightclick` prevent the node formspec from being shown?

Reply:
It still shows

Reply:
Ok, thanks.